docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -107,12 +107,24 @@ When you enter the preceding command, you should see output similar to the follo
 
 [source,subs="+quotes,+attributes"]
 ----
-CONSUMER GROUP ID    ACTIVE MEMBERS    PARTITIONS WITH LAG
--------------------  ----------------  ---------------------
-consumergroup1                     2                     0
-consumergroup2                     1                     0
+Consumer group ID    Active members    Partitions with lag    State
+-------------------  ----------------  ---------------------  -------
+consumergroup1                     2                     0    Empty
+consumergroup2                     1                     0    Stable
 ----
 
+[id="con-consumer-group-state_{context}"]
+==== States of consumer groups
```

Comment:
Adding this section with a level 4 heading makes it subordinate to "Listing consumer groups using the CLI", which isn't the case, because it also describes viewing states in the console. Make it a level 3 heading instead.